spec/support/features/in_person_helper.rb

```diff
@@ -8,6 +8,9 @@ module InPersonHelper
   GOOD_FIRST_NAME = Idp::Constants::MOCK_IDV_APPLICANT[:first_name]
   GOOD_LAST_NAME = Idp::Constants::MOCK_IDV_APPLICANT[:last_name]
   GOOD_DOB = Idp::Constants::MOCK_IDV_APPLICANT[:dob]
+  GOOD_DOB_FORMATTED_EVENT = I18n.l(
+    Date.parse(GOOD_DOB), format: I18n.t('time.formats.event_date')
```

Comment:
It might be clearer to name this `GOOD_DOB_FULL_I18N` or something like that.

Reply:
I added some comments in c8f4f93de9dfd55b16fcc1d4711e4bbed680192e to clarify what the variables hold, but I prefer the current variable name because it matches the I18n naming.